src/rc/start-stop-daemon.c

```diff
@@ -66,8 +70,13 @@ static struct pam_conv conv = { NULL, NULL};
 
 const char *applet = NULL;
 const char *extraopts = NULL;
+#ifdef HAVE_CAP
+const char *getoptstring = "I:KN:PR:Sa:bc:d:e:g:ik:mn:op:s:tu:r:w:x:1:2:3:4:A:" \
```

Comment:
please do not duplicate the entire string.  this will be a nightmare to maintain.

instead, always define the flags.  add a runtime check to the flag parser that errors out if it's used when HAVE_CAP is not defined.